bookstore.py

```diff
@@ -264,9 +266,10 @@ def get_all_books(self):
             for r in rows:
                 book = Book(r['title'], r['author'], r['read'], r['rowid'])
                 books.append(book)
+                count += 1
```

Comment:
SQL has a count function which would be more efficient than getting all the books